Common/Subworlds/MappingNPC.cs

```diff
@@ -0,0 +1,60 @@
+﻿using PathOfTerraria.Common.Systems.Affixes.ItemTypes;
+using SubworldLibrary;
+
+namespace PathOfTerraria.Common.Subworlds;
+
+internal class MappingNPC : GlobalNPC
+{
+	public override void SetDefaults(NPC entity)
+	{
+		if (SubworldSystem.Current is MappingWorld map && map.Affixes is not null)
+		{
+			foreach (MapAffix affix in map.Affixes)
+			{
+				affix.ModifyNewNPC(entity);
+			}
+
+			if (map.Tier > 50)
```

Comment:
This is also slightly confusing, tiers only go up to 11 at this time.
Maybe worth renaming tier to AreaLevel or something?